pubspec.yaml

```diff
@@ -26,6 +26,7 @@ dependencies:
   markdown: ^7.1.1
   mime: ^1.0.0
   olm: ^2.0.2
+  path: ^1.9.0
```

Comment:
I don't think it is worth adding this dependency just for some tests.

Reply:
changed to dev dep